src/ESLintLinter.php

```diff
@@ -22,7 +22,14 @@
   const ESLINT_WARNING = '1';
   const ESLINT_ERROR = '2';
 
+  private $cwd = '';
   private $flags = array();
+  private $realCwd;
+
+  function __construct() {
+    parent::__construct();
+    $this->realCwd = $this->getProjectRoot();
```

Comment:
Is this just to cache the value because `$this->getProjectRoot()`? Could you just call that directly in the one place this is used below?

If not, consider naming this `$projectRoot` instead of `$realCwd`?

Reply:
Fixed/removed.

2 things made this confusing as it was originally:

1. A bad copy & paste from the original implementation in `PrettierLinter.php` left out how it changed with `$this-flags['foo.cwd']`.
2. The original implementation and motivation for this was caching of the `realpath` lookup resolving `$this->cwd` from `$this->getProjectRoot()`, which has also since been removed, but needed to be cleaned up.